### PR TITLE
Release 2.2.1

### PR DESCRIPTION
**Pull request by daschuer**
This is the release PR of a possible 2.2.1 release. 
I think it is reasonable to do it now, to make the 2.1.7 fixes available for all users. 
It also induces the important fixes against performance issue in the library.

### CODE REVIEW

**Comment by Be-ing** [comment]
I do not think it is a good idea to release this until we try to get wider testing for #1953. Unfortunately that is blocked on the build server not working...

**Comment by daschuer** [comment]
Yes, of cause, we need a working CI on Jenkins anyway to publish the release.

File: CHANGELOG [hunk 1]
```diff
@@ -1,3 +1,12 @@
+==== 2.2.1 2019-02-02 ====
+
+* Include all fixed from Mixxx 2.1.7
```
Comment (uklotzde) [inline CHANGELOG]: typo

**Comment by uklotzde** [comment]
@daschuer Please also resolve the small typo and the conflicts. Should we backport the fixes from 2.1.8 before actually releasing 2.2.1?

**Comment by daschuer** [comment]
Yes that was the plane, release 2.18 first and merge it to 2.2 and than release that.

**Comment by daschuer** [comment]
This is now again up to date. Merge?

**Comment by uklotzde** [comment]
#2057 is just a one line change with 2 supporters, but no feedback yet.

**Comment by ywwg** [comment]
> #2057 is just a one line change with 2 supporters, but no feedback yet.

In the past it has been ok to merge in short changes like that that are not possible for other developers to test (because of lack of hardware)

**Comment by daschuer** [comment]
Ready to merge again and actually release 2.2.1

File: CHANGELOG [hunk 2]
```diff
@@ -1,3 +1,14 @@
+==== 2.2.1 2019-02-02 ====
```
Comment (uklotzde) [inline CHANGELOG]: The release date needs to be updated

**Comment by uklotzde** [comment]
Just one minor finding.

Thank you @daschuer for doing all the work preparing this release!

**Comment by daschuer** [comment]
Thank you fro review.

**Pull request merged**